### PR TITLE
Feature/e UI 4200 task error

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4200?filter=-1

### Change description ###
When you claim a task if the api return an error 400 ensure `handleFatalError` function prints the error message. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/work-allocation-2/utils/work-allocation-utils.ts [hunk 1]
```diff
@@ -59,6 +59,29 @@ export const handleFatalErrors = (status: number, navigator: Navigator, fatals?:
   }
 };
 
+export const handleTasksFatalErrors = (status: number, navigator: Navigator, fatals?: FatalRedirect[]): number => {
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/utils/work-allocation-utils.ts:62]: Minor comment - Is this handleTaskFatalErrors necessary as handleFatalErrors is already present in the file and there is one small change for 400 errors?

**Review by connorpgpmcelroy — COMMENTED** [review]
Unsure whether two types of error handling needed here

File: src/work-allocation-2/utils/work-allocation-utils.ts (hunk 1 above)
Comment (kuda1992) [reply src/work-allocation-2/utils/work-allocation-utils.ts:62]: At first glance it's not necessary however `handleFatalErrors` is a generic error function used in multiple places. Changing how we handle errors, just to cater for tasks screen will break the business rules on other screens.

Comment (udaydenduluri33) [reply src/work-allocation-2/utils/work-allocation-utils.ts:62]: Agree with @connorpgpmcelroy  here, The generic error handling should be enough. Are you saying that this change is only for the Tasks Screen?

Comment (kuda1992) [reply src/work-allocation-2/utils/work-allocation-utils.ts:62]: @udaydenduluri33 yes. The solution is reuse `handleFatalErrors` function and add an extra parameter to treat 400 errors from that screen differently or create a new function, I chose the later.

Comment (udaydenduluri33) [reply src/work-allocation-2/utils/work-allocation-utils.ts:62]: Ok makes sense

**Review by udaydenduluri33 — APPROVED** [review]
@kuda1992  thanks for making the changes

**Review by connorpgpmcelroy — APPROVED** [review]

**Pull request merged**